checkov/serverless/runner.py

```diff
@@ -118,7 +118,7 @@ def run(self, root_folder, external_checks_dir=None, files=None, runner_filter=R
                         results = cfn_registry.scan(sls_file, entity, skipped_checks, runner_filter)
                         tags = cfn_utils.get_resource_tags(entity, cfn_registry)
                         for check, check_result in results.items():
-                            record = Record(check_id=check.id, check_name=check.name, check_result=check_result,
+                            record = Record(check_id=check.id, bc_check_id=check.bc_id, check_name=check.name, check_result=check_result,
```

Comment:
i don't recommend changing the constructor of a basic class. let's add a setter.